docs/tutorial.md

```diff
@@ -376,6 +376,37 @@ the scheduler starts up, it detects that it is already running 3
 `data` tasks, and starts 2 more in order to reach the goal state of 5.
 We should observe two more tasks starting, `data-3` and `data-4`.
 
+## Task Configuration management
```

Comment:
Oh yea.  I think we should just remove the content for now, keep it as a PR, and block it on the feature.